### PR TITLE
Added a new remediation to rule rsyslog_files_permissions, now it doe…

### PR DESCRIPTION
**Pull request by rsprudencio**
…sn't touch systemd legacy files to avoid breaking rpm -V systemd.

### CODE REVIEW

File: RHEL/7/templates/static/bash/rsyslog_files_permissions.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+done
+
+for PATH in "${LOG_FILE_PATHS[@]}"
+do
+	# Sanity check - if particular $PATH is empty string, skip it from further processing
+	if [ -z "$PATH" ]
+	then
+		continue
+	fi
+	# In RHEL 7 we have systemd, so it doesn't make sense to patch/change /etc/rc.d/rc.local
```
Comment (yuumasato) [inline RHEL/7/templates/static/bash/rsyslog_files_permissions.sh]: As in the shared remediation, I would put the link to the bugzilla here.

**Comment by rsprudencio** [comment]
@yuumasato I added the BZ link to the comment :smile:

File: RHEL/7/templates/static/bash/rsyslog_files_permissions.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# platform = multi_platform_rhel
```
Comment (yuumasato) [inline RHEL/7/templates/static/bash/rsyslog_files_permissions.sh]: And platform should be Red Hat Enterprise Linux 7.
Sorry that I didn't notice it in previous review.

Comment (rsprudencio) [inline RHEL/7/templates/static/bash/rsyslog_files_permissions.sh]: Good catch :)

Updated

**Comment by rsprudencio** [comment]
@yuumasato Good catch :)

Updated

**Pull request merged**